MediaRecorder.html

```diff
@@ -340,149 +340,87 @@
         </section>
 
 
-   <section id="error-handling">
-      <h2>Error Handling</h2>
-
-      <section id="general-principles">
-        <h3>General Principles</h3>
-
-        <p>Errors are indicated in two ways: exceptions and objects passed to
-        error callbacks. In the former case, a <a href="http://www.w3.org/TR/2012/WD-dom-20121206/#exception-domexception">DOMException</a> is raised (see [[!DOM4]]).
-        An exception <em title="must" class="rfc2119">must</em> be thrown when the
-        error can be detected at the time that the call is made. 
-        In all other cases, a <code>MediaRecorderErrorEvent</code>  <em title="must" class="rfc2119">must</em> be raised. 
-        The error name <em title="must" class="rfc2119">must</em> be picked from
-         the <code>RecordingErrorName</code>
-        enums.  If recording has been started and not yet stopped when the error occurs, then after raising the error, the UA <em title="must" class="rfc2119">must</em>
-        raise a  dataavailable event, containing any data that it has gathered,
-        and then a stop event. The UA  <em title="may" class="rfc2119">may</em> set platform-specific
-        limits, such those for the minimum and maximum Blob size that it will support, or the number of
-        Tracks it will record at once.  It <em title="must" class="rfc2119">must</em> signal a fatal
-        error if these limits are exceeded. </p>
- </section>
-
-
+  <section id="error-handling">
+    <h2>Error Handling</h2>
 
+    <section id="general-principles">
+      <h3>General Principles</h3>
 
+      <p>Errors are indicated via <a href="http://www.w3.org/TR/2012/WD-dom-20121206/#exception-domexception">DOMException</a>s raised (see [[!DOM4]]). An exception <em title="must" class="rfc2119">must</em> be thrown when the error can be detected at the time that the call is made. If recording has been started and not yet stopped when the error occurs, then after raising the error, the UA <em title="must" class="rfc2119">must</em> raise a  dataavailable event, containing any data that it has gathered, and then a stop event. The UA  <em title="may" class="rfc2119">may</em> set platform-specific limits, such those for the minimum and maximum Blob size that it will support, or the number of Tracks it will record at once.  It <em title="must" class="rfc2119">must</em> signal a fatal error if these limits are exceeded. </p>
```

Comment:
DOMException is defined by WebIDL, so https://heycam.github.io/webidl/#idl-DOMException is the right reference. DOM itself (https://dom.spec.whatwg.org) references WebIDL for this.